guides/common/modules/proc_invalidating-your-own-jwts.adoc

```diff
@@ -0,0 +1,34 @@
+[id="invalidating-your-own-jwts"]
+= Invalidating your own JWTs
+
+You can invalidate all registration JSON Web Tokens of the current user.
+
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-invalidating-your-own-jwts[].
+To use the API, see the xref:api-invalidating-your-own-jwts[].
+
+.Prerequisites
+* You are logged in to the {ProjectWebUI}.
+
+.Procedure
+. In the {ProjectWebUI}, click the user menu in the top bar and select *My Account*.
+. Select the *Registration Tokens* tab.
+. Click *Invalidate JWTs*.
+. In the confirmation window, click *Confirm*.
+
+.Verification
+* The {ProjectWebUI} displays the following message: `Successfully invalidated registration tokens.`
+
+[id="cli-invalidating-your-own-jwts"]
+.CLI procedure
+* Invalidate all your registration tokens by running Hammer:
++
+[options="nowrap" subs="+quotes,attributes,verbatim"]
+----
+$ hammer registration-tokens invalidate
+----
+
+[id="api-invalidating-your-own-jwts"]
+.API procedure
+* Use the `DELETE /api/registration_tokens` resource.
```

Comment:
wrong endpoint, user has to use  `DELETE api/users/:self_user_id/registration_tokens` for this process :)